kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java

```diff
@@ -35,6 +35,8 @@
 
     void completeTask(String containerId, Long taskId, String userId, Map<String, Object> params);
 
+    void claimStartCompleteTask(String containerId, Long taskId, String userId, Map<String, Object> params);
+
```

Comment:
true, it does that at the moment but if we change the internals then it might not be as exact any more ;)